confirmation_rule.py

```diff
@@ -162,7 +168,8 @@ def get_latest_confirmed(store) -> Root:
     confirmed_block_slot = store.blocks[confirmed_root].slot
     unrealized_justified_block_slot = store.blocks[store.unrealized_justified_checkpoint.root].slot
     if (confirmed_block_slot < unrealized_justified_block_slot
-        and store.unrealized_justified_checkpoint.epoch == current_epoch):
+        and store.unrealized_justified_checkpoint.epoch == current_epoch
+        and store.prev_slot_justified_checkpoint.epoch + 1 >= current_epoch):
```

Comment:
```suggestion
        and store.prev_slot_justified_checkpoint.epoch + 1 == current_epoch):
```
maybe this way would be clearer?